src/reducers/recent-projects.js

```
@@ -0,0 +1,68 @@
+    switch (action.type) {
+    case LOAD_RECENT_PROJECTS:
+        return {
+            ...state,
+            projects: action.projects || []
+        };
+    case ADD_RECENT_PROJECT:
+        if (!action.project) {
+            return state;
+        }
+        // Add to front, maintain max 5, remove duplicates by name
+        const existingProjects = state.projects.filter(p => p.name !== action.project.name);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)